pom.xml

```
@@ -1,959 +1,948 @@
+        <tomcat.version>9.0.12</tomcat.version>
+        <thorntail.version>2.3.0.Final</thorntail.version>
+        <payara_domain>domain1</payara_domain>
+        <parsson.version>1.1.1.payara-p1</parsson.version>
+        <jersey.version>3.1.0-M7.payara-p1</jersey.version>
+        <jakartaee.api.version>10.0.0</jakartaee.api.version>
+        <servlet-api.version>6.0.0</servlet-api.version>
+        <jakarta.annotation.version>2.1.1</jakarta.annotation.version>
+        <jakarta.xml.bind-api.version>4.0.0</jakarta.xml.bind-api.version>
+        <jaxb-runtime.version>4.0.0</jaxb-runtime.version>
+        <jakarta.ws.rs-api.version>3.1.0</jakarta.ws.rs-api.version>
+        <yasson.version>3.0.0-RC2</yasson.version>
+        <org.glassfish.tyrus.version>2.1.0.payara-p1</org.glassfish.tyrus.version>
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>central</id>
+            <name>Central Repository</name>
+
+            <url>https://repo.maven.apache.org/maven2</url>
+
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+
+        <repository>
+            <id>payara-milestones</id>
+            <name>Payara Milestones</name>
+            <url>https://raw.github.com/payara/Payara_PatchedProjects/master</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+
+        <repository>
+            <id>ossrh</id>
+            <name>Sonatype-snapshot</name>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
+
+        <repository>
+            <id>jvnet-nexus-staging</id>
+            <name>Java.net Staging Repositories</name>
+            <url>https://maven.java.net/content/repositories/staging</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+
+        <repository>
+            <id>jvnet-nexus-promoted</id>
+            <name>Java.net Promoted Repositories</name>
+            <url>https://maven.java.net/content/repositories/promoted/</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+
+        <repository>
+            <id>staged-jakarta-releases</id>
+            <url>https://jakarta.oss.sonatype.org/content/groups/staging/</url>
+        </repository>
```

Missing the following repository for patched Payara artefacts (e.g. Jersey 3.1.0-M7.payara-p1):

```suggestion
        </repository>
        <repository>
            <id>payara-nexus-artifacts</id>
            <url>https://nexus.payara.fish/repository/payara-artifacts</url>
        </repository>
```